packages/utils/src/CypressUtils/PaginationUtils.ts

```diff
@@ -0,0 +1,37 @@
+//need the following line to explain typescript that cy is = to cypress
+/// <reference types="cypress" />
+// @ts-nocheck
```

Comment:
Yep, that was a solution " to make it work" and develop the MVP. It was my plan to go back and remove them + add types

Reply:
In these tests it looks like there is not much overhead with TS, usually you know what to add just with hints that TS raises on errors. Define you MVP exactly: either we don't work with TS and then change the extension, or start using it now. I would opt for the second, because in either way we want to work on tests further

Reply:
Can we ignore TS in that case? I was writing tests for frontend-components in JS. I am of the opinion that it's just enough to write tests with JS, whereas for the source code the choice is definitely for TS for the sake of easier refactor and stuff

Reply:
Sure, will do that then